net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml

```diff
@@ -1,20 +1,28 @@
 <model>
-  <mount>//OPNsense/MDNSRepeater</mount>
-  <version>1.0.1</version>
-  <description>mdns-repeater settings</description>
-  <items>
-    <enabled type="BooleanField">
-      <Default>0</Default>
-      <Required>Y</Required>
-    </enabled>
-    <enablecarp type="BooleanField">
-      <Default>0</Default>
-      <Required>Y</Required>
-    </enablecarp>
-    <interfaces type="InterfaceField">
-      <Default>lan</Default>
-      <Required>Y</Required>
-      <Multiple>Y</Multiple>
-    </interfaces>
-  </items>
+    <mount>//OPNsense/MDNSRepeater</mount>
+    <version>1.0.1</version>
+    <description>mdns-repeater settings</description>
+    <items>
+        <enabled type="BooleanField">
+            <Default>0</Default>
+            <Required>Y</Required>
+        </enabled>
+        <enablecarp type="BooleanField">
+            <Default>0</Default>
+            <Required>Y</Required>
+        </enablecarp>
+        <interfaces type="InterfaceField">
+            <Default>lan</Default>
+            <Required>Y</Required>
+            <Multiple>Y</Multiple>
+        </interfaces>
+        <blacklist type="NetworkField">
```

Comment:
let's please call this "blocklist"

Reply:
But its called blacklist in source:

https://github.com/geekman/mdns-repeater/releases/tag/1.11

Reply:
Not our problem. 😊

Reply:
@fichtner The deed has been done.